test/tests/queries/totalCount.test.js

```diff
@@ -160,19 +160,34 @@ describe('graphql - queries with totalCount', () => {
     `
     const data = {
       AdminService: {
-        Authors: null
+        Authors: {
+          totalCount: 4,
+          nodes: [
+            {
+              name: 'Edgar Allen Poe',
+              books: {
+                totalCount: null, // REVISIT: expected 2
```

Comment:
the totalCount is probably being returned as `books@odata.count`. As defined [here](https://github.com/cap-js/cds-dbs/blob/main/db-service/lib/cqn2sql.js#L255)

Reply:
I'll add a BLI for us. Seems very "odata'ish" :)